code/modules/cm_preds/yaut_weapons.dm

```diff
@@ -278,6 +280,10 @@
 	return TRUE
 
 /obj/item/weapon/yautja/combistick/proc/setup_chain(mob/living/user)
+	give_action(user, /datum/action/predator_action/bracer/combistick)
+	add_verb(user, /mob/living/carbon/human/proc/call_combi)
+	linked_to = user
+
 	var/list/tether_effects = apply_tether(user, src, range = 6, resistable = FALSE)
```

Comment:
This from dropping a combistick results in `/datum/effects/tethering/proc/set_tethered(datum/effects/tethered/T)` throwing `Exception has occurred: Cannot execute null.beam().` when self destructing
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/327ea846-290a-4b72-8942-b07950c0b48d)